style.css

```diff
@@ -0,0 +1,818 @@
+    flex: .44;
+  }
+
+  .character__item { padding-bottom: 8px; }
+
+  .character__icon_left { width: 31%; }
+
+  .character_left { padding-left: 4px; }
+
+  .character_right { padding-right: 7px; }
+
+  .section-main__block-buttons { width: 96%; }
+
+  .button__icon { width: auto; }
+
+  .section-main__button-order {
+    height: 37px;
+    padding-right: 12px;
+    flex: .74;
+  }
+}
+
+/* Адаптив второй секции */
+
+/* Планшет */
+@media (max-width: 768px) {
+  .section-desc {
+    display: block;
+    width: auto;
+    height: auto;
+    background: 0;
+
+  }
+  .section-desc__wrap_up {
+    height: 486px;
+    padding-top: 45px;
+    background: url(./Project/item_interior.svg) no-repeat;
+    background-size: contain;
+    background-position: 0 100%;
+  }
+
+  .section-desc__title {
+    width: 94%;
+    text-align: right;
+    padding-left: 0;
+  }
+
+  .section-desc__text {
+    display: flex;
+    flex-wrap: wrap;
+    width: auto;
+    height: 485px;
+    padding-top: 0;
+  }
+
+  .text__wrap {
+    width: 45.5%;
+    min-width: 268px;
+    margin: 0;
+    padding-top: 57px;
+    padding-left: 33px;
+  }
+
+  .text__title { margin-bottom: 17px; }
+
+  .text__content {
+    width: 90%;
+    height: 220px;
+    margin: 0;
+    padding-left: 2px;
+    letter-spacing: .6px;
+  }
+
+  .text__character {
+    height: 243px;
+    margin: 0;
+    padding-top: 57px;
+    padding-left: 69px;
+  }
+
+  .text__item_bag,
+  .text__small-wifi { padding-bottom: 0; }
+
+  .text__small-screen,
+  .text__small-bag { padding: 10px 0 0 2px; }
+  .text__small-screen { margin-right: 20px; }
+  .text__small-bag { margin-right: 25px; }
+
+  .text__small-wifi { margin-right: 17px; }
+
+  .buttons {
+    align-items: center;
+    margin: 0 auto;
+    padding-bottom: 7px;
+  }
+
+  .button__link_info { color: #fff; }
+}
+
+@media (max-width: 650px) {
+  .text__wrap,
+  .text__character { padding-left: 20px; }
+
+  .text__small-screen { margin-right: 13px; }
+
+  .text__small-wifi { margin-right: 8px; }
+
+  .text__small-bag { margin-right: 12px; }
+
+  .text__icon { max-width: 80%; }
+
+  .button__item_icon { width: auto; }
+}
+
+@media (max-width: 525px) {
+  .section-desc__title,
+  .text__character,
+  .buttons { display: none; }
+
+
+  .section-desc__wrap_up {
+    background-size: cover;
+    background-position: 0 5px;
+  }
+
+  .section-desc__wrap_up { padding-top: 0; }
+
+  .section-desc__text { height: 305px; }
+
+  .text__wrap {
+    width: auto;
+    padding-top: 17px;
+  }
+
+  .text__title { margin-bottom: 9px; }
+}
+
+/* Телефон */
+@media (max-width: 375px) {
+  .section-desc__title { display: none; }
+
+  .section-desc__wrap_up { background-position: -35px 5px; }
+
+  .section-desc__text { min-width: 0; }
+
+  .text__wrap { padding-left: 0; }
+
+  .text__title {
+    padding-left: 26px;
+    letter-spacing: .6px;
+  }
+
+  .text__content {
+    width: 85%;
+    height: 238px;
+    margin: auto;
+    letter-spacing: .6px;
+  }  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Project_page/pullRequest?prid=5536520)